add-on/chrome/bootstrap.js

```diff
@@ -637,6 +638,10 @@ var WindowListener = {
 
         let class_name = "clicked";
         let cursor = document.getElementById("loop-remote-cursor");
+        if (!cursor) {
+          return;
+        }
+
```

Comment:
Maybe update the jsdoc commentary to be explicit that this only adds the ripple if the cursor is already present, and does nothing otherwise.